lib/ChangeDueDateDialog/ChangeDueDate.js

```diff
@@ -249,7 +250,7 @@ class ChangeDueDate extends React.Component {
           { loans.length > 1 ?
             <SafeHTMLMessage
               id="stripes-smart-components.cddd.itemsSelected"
-              values={{ count: loans.length }}
+              values={{ count: Object.values(loanSelection).filter(Boolean).length }}
```

Comment:
Does it make sense to move the calculation of length into a variable?

Reply:
Sounds good!